OpenSim/Common/Exception.h

```diff
@@ -67,14 +91,29 @@ class OSIMCOMMON_API Exception  : public std::exception {
     std::string _file;
     /** Line number at which the error occurred. */
     int _line;
+    /** Delimiter/separator used to separate different parts of the message.  */
+    static const std::string _msgSeparator;
 
 //=============================================================================
 // METHODS
 //=============================================================================
 public:
     // CONSTRUCTORS
-    Exception(const std::string &aMsg="",const std::string &aFile="",int aLine=-1);
+    Exception(const std::string &aMsg="",
```

Comment:
It would be nice to add a comment to state that this constructor is for backwards compatibility. I also noticed that its implementation has not been updated to delegate to the new Constructor. I would expect it to use `Exception(file, line, func)` and then append its message. That way the format of old Exceptions would match the new.